ci/scripts/lint-java.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -euo pipefail
+
+function print() {
+	GREEN='\033[0;32m'
+  NC='\033[0m'
+  echo
+	echo -e "${GREEN}${1}${NC}"
+}
+
+dirs=("$(find . -name "*-java" -type d -not -path '*/.*')")
+for dir in $dirs; do
+  print "Linting $dir"
+  pushd $dir
+
+  if [[ -f "pom.xml" ]]; then
+    print "Running Maven Build"
+    mvn clean package
```

Comment:
worth updating to the use the maven wrapper approach as well

Reply:
Better idea:   send `mvn` to the bin entirely...